README.adoc

```diff
@@ -366,10 +364,11 @@ interface for span creation and customization, including setting timestamps.
 
 == Testing the services
 
-A few tests are included for you to test the basic functionality of the services. If a test
-failure occurs, then you might have introduced a bug into the code. These tests will run automatically
-as a part of the Maven build process when you run `mvn install`. You can also run these tests separately
-from the build by using the `mvn verify` command, but first make sure that the servers are stopped.
+A few tests are included for you to test the basic functionality of the services. not including the
+correctness of the traces. If a test failure occurs, then you might have introduced a bug into the code.
```

Comment:
I think we just have a typo here: "A few tests are included for you to test the basic functionality of the services. **not including the correctness of the traces.**"
Do we just want to delete "not including the correctness of the traces."?
Or maybe..."A few tests are included for you to test the basic functionality of the services **but do not include testing the correctness of the traces.**"

Reply:
Yes, this is a typo. What about "A few tests are included for you to test the basic functionality of the services, but they do not include testing the correctness of the traces."

Reply:
Yes, your suggestion sounds good! Thank you!